admin/actions.php

```diff
@@ -72,60 +72,34 @@ function hmbkp_request_do_backup() {
 		check_admin_referer( 'hmbkp_run_schedule', 'hmbkp_run_schedule_nonce' );
 	}
 
-	// Fixes an issue on servers which only allow a single session per client
-	session_write_close();
+	HM\BackUpWordPress\Path::get_instance()->cleanup();
 
-	// We want to display any fatal errors in this ajax request so we can catch them on the other side.
-	error_reporting( E_ERROR | E_USER_ERROR | E_CORE_ERROR | E_COMPILE_ERROR | E_RECOVERABLE_ERROR );
```

Comment:
We'll lose access to these errors, we should open a new ticket to add this functionality back in, probably via `register_shutdown_function`